### ISSUE TITLE
-bash: cobra-cli: command not found

### ISSUE THREAD

**Issue by Damantino** [spf13/cobra-cli#105]
Hi, 

I'm running on command not found when installing the cli.

Steps I did:

1. Install cli by running: `go install github.com/spf13/cobra-cli@latest`
2. Making sure my `~/.bashrc` contains $GOPATH: `exportPATH="$PATH:/Users/daniperez/.local/bin:$GOPATH/bin"`
3. `source ~/.bashrc`
4. Try cobra-cli command

I double checked and cobra-cli is present under `go/bin`: cobra-cli	controller-gen	gopls
If it helps I'm running on a macOs Monterey v12.7.1

Am I doing something wrong?

**Comment by marckhouzam** [comment]
Does it run if you do `$GOPATH/bin/cobra-cli -h`?

**Comment by Damantino** [comment]
Nope :(

Says: `bash: /bin/cobra-cli: No such file or directory`

**Comment by marckhouzam** [comment]
Your $GOPATH variable is not set.

**Comment by Damantino** [comment]
When I do go env it outputs `GOPATH='/Users/********/go/bin'`
When I do `echo $GOPATH ` it outputs `/Users/*******/go`

Does that mean it's not set? If so, how do I set it?

**Comment by Damantino** [comment]
Ok now I got it after googling a lot.

I succeded on my quest to use cli by editing `bash_profile`

Steps:

1. edit bash_profile. I did it with command `vim ~/.bash_profile`
2. Insert at end of file. `export GOPATH=$HOME/go` and `export PATH=$GOPATH/bin:$PATH`
3. Restart bash with `source ~/.bash_profile`

I know it ay seem obvious for lots of you guys but this worked for me. We can close this issue now, thanks for your time @marckhouzam
